### PR TITLE
🤑 Create deals

### PR DESCRIPTION
**Pull request by erwagasore**
- Created deals for a given user in a given country
- Corrected issues withing `awaremodel` that was causing the coverage not to be `100%`
- Fully tested both `deals` and `awaremodel`

### CODE REVIEW

**Comment by erwagasore** [comment]
screen shot 2018-10-06 at 22 12 30

@kenessajr please review

File: api/app/users/test_users.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def test_users_routes(self, db_session, credentials):
         assert resp.status_code == 400
 
         # post all required fields for the user
-        data = dict(username='foo', email='foo@bar.baz', password='foo')
+        data = dict(username='foo', email='foo@bar.baz', password='foo', force=True)
```
Comment (kenessajr) [inline api/app/users/test_users.py]: @erwagasore what does the `force attr` do?

Comment (erwagasore) [reply api/app/users/test_users.py]: `force` is an attribute for making a user a `super user`

Comment (erwagasore) [reply api/app/users/test_users.py]: The reason I have added it there is to be able to test the import and export of a boolean value

Comment (kenessajr) [reply api/app/users/test_users.py]: Awesome.

**Pull request merged**